cmds/kexec/kexec_linux.go

```diff
@@ -45,6 +48,46 @@ func registerFlags() *options {
 	return o
 }
 
+type loader interface {
+	Load(path, cmdLine, initramfs string) error
```

Comment:
look at pkg/boot. I started moving these kinds of things into there. it should have an interface like this, actually.